components/your-history/ActivityCard.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { format } from 'date-fns';
+import Card from '@material-ui/core/Card';
+import Chip from '@material-ui/core/Chip';
+import Divider from '@material-ui/core/Divider';
+import Grid from '@material-ui/core/Grid';
+
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import { ActivityPropTypes } from './PropTypes';
+
+function getFormattedDateCompleted(timestamp) {
+  const date = timestamp.toDate();
+  return format(date, 'MMMM do');
+}
+function getFormattedDateEntered(timestamp) {
+  const date = timestamp.toDate();
+  return format(date, 'P');
+}
+
+const useStyles = makeStyles(theme => ({
+  card: {
+    minWidth: 275,
+    padding: theme.spacing(2),
+  },
+  divider: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
+  group: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
+  label: {
+    fontWeight: theme.typography.fontWeightMedium,
+  },
+  chip: {
+    marginRight: theme.spacing(1),
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+  },
+  description: {
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(4),
+  },
+}));
+
+function ActivityCard(props) {
+  const classes = useStyles();
+
+  const {
+    timestamp,
+    activityTypeLabel,
+    description,
+    briefDescription,
+    dateCompleted,
+    timeSpentInMinutes,
+    difficultyLevel,
+    activityFeeling,
+    whyIfeelThisWay,
+  } = props;
+  return (
+    <Card className={classes.card}>
+      <CardContent>
+        <div className={classes.header}>
+          <Typography variant="body1" noWrap style={{ maxWidth: '50%', fontWeight: 500 }}>
```

Comment:
<img width="1074" alt="Screen Shot 2019-12-07 at 11 46 49 AM" src="https://user-images.githubusercontent.com/44956/70453720-7b8c4c80-1a66-11ea-9f39-c019de12f738.png">

Does this need to be limited to 50%? Truncates longer type names.

Reply:
I briefly brought it up to Alex, but can easily push it to 75%

Reply:
75% seems good.